finat/physically_mapped.py

```diff
@@ -163,6 +201,14 @@ def jacobian_at(self, point):
         :returns: A GEM expression for the Jacobian, shape (gdim, tdim).
         """
 
+    @abstractmethod
+    def detJ_at(self, point):
+        """The determinant of the jacobian of the physical coordinates at a point.
+
+        :arg point: The point in reference space to evaluate the Jacobian determinant.
+        :returns: A GEM expression for the Jacobian determinant.
+        """
+
```

Comment:
@rckirby: I made the same mistake earlier this week, but if you extends this interface, then you ought to merge the corresponding TSFC change (which provides the implementation) together with the FInAT change, otherwise Firedrake will break.